modules/data.land/R/gSSURGO_Query.R

```diff
@@ -0,0 +1,77 @@
+############ Retrives soil data from gssurgo
+#' This function queries the gSSURGO database for a series of map unit keys
+#'
+#' @param mukeys map unit key from gssurgo
+#'
+#' @return a dataframe with soil properties. units can be looked up from database documentation
+#' @export
+#'
+#' @details 
+#' 
+#' Full documention of available tables and their relationships can be found here https://sdmdataaccess.nrcs.usda.gov/QueryHelp.aspx
+#' There have been occasions where NRCS made some minor changes to the structure of the API which this code is where those changes need
+#' to be implemneted here.
+gSSURGO.Query<-function(mukeys=2747727){
+  require(XML)
```

Comment:
Don't use `require` or `library` in functions -- either use `@import`/`@importFrom` roxygen tags, or, preferably, call external package functions through `package::function` syntax.